### PR TITLE
Display Menu

### PR DESCRIPTION
**Pull request by astroboyReloaded**
**For this exercise, we did the following:** 📑

- Create and link `menu.js` file. 🔌
- Select **DOM** elements with `.getElementById`, `getElementsByClassName`, and `.querySelector`. ⚓ 
- Write `.toggle` function. 🚦 
- Add event listeners with `element.onclick`. 👂 
- Style the `<nav>` element. 🎨 

😎

### CODE REVIEW

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @astroboyReloaded @muhammedb8826 👋, 

Good job so far!

### Highlights
- All linters checks passed ✔️ 
- Clean _**JavaScript**_ codes ✔️ 
- _**Hamburger**_ button functions properly ✔️ 
- Links in the mobile menu direct me to the correct part of the page ✔️ 
- Closing icon closes the mobile menu ✔️ 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- Kindly set up the Eslint configuration for this project. The configuration checks our JavaScript codes and ensures we write clear, easy-to-read and maintainable JavaScript codes. 👍 💻 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please, remember to tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: scripts/menu.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+const menuBtn = document.getElementById('menu-toggle');
+const nav = document.querySelector('nav');
```
Comment (joskalenda) [inline scripts/menu.js:2]: - [ ] Well done @astroboyReloaded, you have at least created the toggle functions but .... Either it is not how you should render your projects, I remark the first issue is 

1. Nothing happens when you click the `see project button` button. fix this issue as we go
2. You are still using your static project data on the HTML page, see ⬇️ 


but referring to the project requirement below, it is now how you should present the projects, instead, they should be dynamic(creating using js, and rendering on page load)



- Pease ensures you remove all project static codes on the page and use js array to store your project data.


- [ ] THIS IS THE RECORD

I have made sure to clone your project as well as use the online version checking, but still the same thing

**Review by joskalenda — CHANGES_REQUESTED** [review]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @astroboyReloaded , |
|Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there! |
|  |

## Highlights

- [ ]... ✅ Great design
- [ ].. ✅ Green linters

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: scripts/menu.js (hunk 1 above)
Comment (astroboyReloaded) [reply scripts/menu.js:2]: Sorry, @joskalenda. I submitted the project to the wrong Request. 😕 
Thank you, though!

**Review by huseyinbiyik — APPROVED** [review]
Hi team,

## ✔️ Status: APPROVED 🏆 🎉 🎊

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text



## Highlights
- Nav functioning perfectly 👌
- Collab work 😎😎
- Cool animations 📱
- EsLint set up well and checks pass 🟢
- Github flow followed 👌
- PR description is descriptive 📝

## Optional suggestions
-  Please make sure your commit messages are meaningful and descriptive for your next commits. Use imperative tense and capitalize the first word. For example, 
👌"Add logo to navigation bar"
👌"Fix close button functionality on mobile navbar"
❌ "Added logo to navigation bar"
❌ "Added HTML"

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**